app/views/finders/_hidden_clearable_facet.html.erb

```diff
@@ -1,12 +1,12 @@
-<div id="<%= hidden_clearable_facet.key %>" style="display: none">
-<%
-  key = hidden_clearable_facet.key
-  values = params[key]
-  key = "#{key}[]" if values.is_a?(Array)
--%>
-<% [values].flatten.each_with_index do |value, index| -%>
+<div id="<%= hidden_clearable_facet.key %>">
+  <%
+    key = hidden_clearable_facet.key
+    values = params[key]
+    key = "#{key}[]" if values.is_a?(Array)
+  -%>
+  <% [values].flatten.each_with_index do |value, index| -%>
 
 
-  <%= text_field_tag key, value, id: "#{hidden_clearable_facet.key}_#{index}" %>
```

Comment:
Only thing about this that I'm unsure of is this change - it's going from a text file to a hidden input. I don't know if that'll have any impact on anything? I assume not, but I wanted to check, not sure where this is.

An alternative might be to replace `style="display: none"` with `hidden`?